solidity/ecdsa/test/WalletRegistryGovernance.test.ts

```diff
@@ -2294,7 +2294,7 @@ describe("WalletRegistryGovernance", async () => {
         await restoreSnapshot()
       })
 
-      it("should not update the wallet owner", async () => {
+      it("should not update the reimbursement pool", async () => {
         expect(await walletRegistry.walletOwner()).to.be.equal(
```

Comment:
We are still checking the wallet owner.

Reply:
🤦 [63ab696](https://github.com/keep-network/keep-core/pull/2856/commits/63ab69687e768fe377bd9ed6183a8188c70a49a0)